app/src/test/java/org/feature/fox/coffee_counter/di/services/network/ApiServiceTest.kt

```diff
@@ -170,7 +172,7 @@ class ApiServiceTest {
 
         mockWebServer.enqueue(response)
 
-        val actualResponse = apiService.deleteItem(itemID)
+        val actualResponse = apiService.deleteItem(bearerToken, itemID)
```

Comment:
The test cases should also assert if the authorization header was send correctly to mock webserver